src/settings.py

```diff
@@ -18,6 +18,9 @@ class GlobalSettings:
     cover_block_widths = [0.1, 0.07]
     cover_target_widths = [0.05, 0.03]
 
+    # cover_multistep_options parameters
+    action_space_limits = [-np.inf, np.inf]
```

Comment:
can you change this to `cover_multistep_options_action_space_limits`? or if that's too long, `cover_multistep_action_limits`

Reply:
Will do the latter.